src/igvfd/schemas/changelogs/assay_term.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`assay_term.json`*
 
+### Minor changes since schema version 9
+
+* Extend `preferred_assay_title` enum list to include `varACCESS`.
```

Comment:
Minor typo, in assay term the property is plural `preferred_assay_titles`

Reply:
Updated.